frontend/src/components/HacknightsParticipants.vue

```diff
@@ -65,6 +65,7 @@
               <v-icon>mdi-account-outline</v-icon>
             </v-list-item-avatar>
             <v-list-item-title v-text="item.github"></v-list-item-title>
+            <v-btn v-on:click="onDeleteParticipant(item)">Delete</v-btn>
```

Comment:
Could we consider sth more graphically aesthetic than DELETE? :D We have [font awsome oboard](https://fontawesome.com/v5.15/icons?d=gallery&p=2&q=delete) (e.g. see the exit button in `ModalContent.vue`) so we could use sth like these:
* https://fontawesome.com/v5.15/icons/user-times?style=solid
* https://fontawesome.com/v5.15/icons/backspace?style=solid

Reply:
> Could we consider sth more graphically aesthetic than DELETE? :D We have [font awsome oboard](https://fontawesome.com/v5.15/icons?d=gallery&p=2&q=delete) (e.g. see the exit button in `ModalContent.vue`) so we could use sth like these:
> 
>     * https://fontawesome.com/v5.15/icons/user-times?style=solid
> 
>     * https://fontawesome.com/v5.15/icons/backspace?style=solid

@OtisRed agreed - these DELETE button is kind of a psycho :D